tests/test_statsummary.py

```diff
@@ -0,0 +1,35 @@
+from CovidTracker.calculate_stat_summary import calculate_stat_summary
+from CovidTracker.get_covid_data import get_covid_data
+import pandas as pd
+import pytest
+
+
+def test_calculate_stat_summary():
+    """Test the calculate_stat_summary() function"""
+
+    input = get_covid_data()
+   
+    # Test output type
+    assert isinstance(calculate_stat_summary(input), pd.DataFrame)
+    
+    # Test output size
+    assert calculate_stat_summary(input).shape == (14, 4)
+
+    # Test output column names
+    assert 'date_report' in calculate_stat_summary(input).columns
+    assert 'province' in calculate_stat_summary(input).columns
+    assert 'cumulative_cases' in calculate_stat_summary(input).columns
+    assert 'cases' in calculate_stat_summary(input).columns
+    
+def test_calculate_stat_summary_errors():
```

Comment:
I think you also need to have tests that test the output of your function is correct, and not only test that the input types are correct